src/lib/middleware/demo-authentication.ts

```diff
@@ -19,14 +19,26 @@ function demoAuthentication(
 ): void {
     app.post(`${basePath}/auth/demo/login`, async (req: IAuthRequest, res) => {
         let { email } = req.body;
-        email = flagResolver.isEnabled('encryptEmails', { email })
-            ? encrypt(email)
-            : email;
+        let user: IUser;
+
         try {
-            const user = await userService.loginUserWithoutPassword(
-                email,
-                true,
-            );
+            if (!authentication.authDemoAllowAdminLogin && email === 'admin') {
+                return res.status(403)
+                    .json({ error: `Admin login not enabled for demo auth type`})
+                    .end;
+            } else if (authentication.authDemoAllowAdminLogin && email === 'admin') {
```

Comment:
This is the other part of the auth status change that we've discussed previously.

```suggestion
            if (authentication.authDemoAllowAdminLogin && email === 'admin') {
```